### PR TITLE
Add detect for the GetBattery API

### PR DESCRIPTION
**Pull request by abrad1212**

### CODE REVIEW

File: feature-detects/battery.js [hunk 1]
```diff
@@ -16,5 +16,5 @@
 Detect support for the Battery API, for accessing information about the system's battery charge level.
 */
 define(['Modernizr', 'prefixed'], function(Modernizr, prefixed) {
-  Modernizr.addTest('batteryapi', !!prefixed('battery', navigator), {aliases: ['battery-api']});
+  Modernizr.addTest('batteryapi', !!prefixed('getBattery', navigator), {aliases: ['battery-api']});
```
Comment (patrickkettner) [inline feature-detects/battery.js]: as both shipped in browsers, we want to test for both.

Comment (abrad1212) [reply feature-detects/battery.js]: @patrickkettner Ok so a new file should be created?

**Comment by patrickkettner** [comment]
sorry, I wasn't clear. Not a new detect, both versions of the API should be detected by the same `battery` detect

**Comment by abrad1212** [comment]
@patrickhlauke Ok. I think I understand, I just don't know how to implement it (I'm fairly new to this library). :P

**Comment by patrickkettner** [comment]
@abrad1212 you would just do both detects, with an `||` in between

```js
!!prefixed('battery', navigator) || !!prefixed('getbattery', navigator)
```

you would also want to change the file and detect name back to just `battery`

File: feature-detects/battery.js [hunk 2]
```diff
@@ -16,5 +16,5 @@
 Detect support for the Battery API, for accessing information about the system's battery charge level.
 */
 define(['Modernizr', 'prefixed'], function(Modernizr, prefixed) {
-  Modernizr.addTest('batteryapi', !!prefixed('battery', navigator), {aliases: ['battery-api']});
+  Modernizr.addTest('batteryapi', !!prefixed('battery', navigator) || !!prefixed('getbattery', navigator), {aliases: ['battery-api']});
```
Comment (patrickkettner) [inline feature-detects/battery.js]: sorry, didn't notice before. But this should be `getBattery`, not `getbattery`

Comment (abrad1212) [reply feature-detects/battery.js]: Oh lol, I should have known. :P Thanks

**Pull request merged**